CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 13.0.0
+
+### Features
+
+### Fixes
+
+### Chore & Maintenance
```

Comment:
should we keep the template 13.x on top?

Reply:
13.x is for changes that have not been published yet. I feel like since we are going to release 13.0.0 the section of 13.x would be empty